CMakeLists.txt

```
@@ -78,6 +78,8 @@ whip_add_function(whip_functions_section device_synchronize "" DeviceSynchronize
 whip_add_function(whip_functions_section event_create "event_t* event" EventCreate)
 whip_add_function(whip_functions_section event_create_with_flags "event_t* event;unsigned flags" EventCreateWithFlags)
 whip_add_function(whip_functions_section event_destroy "event_t event" EventDestroy)
+whip_add_function(whip_functions_section event_elapsed_time "float* millisecond;event_t startEvent;event_t stopEvent" EventElapsedTime)
```

I'm changing the parameters to snake case just to match the rest of `whip`. Obviously this does not affect your usage in any way.
```suggestion
whip_add_function(whip_functions_section event_elapsed_time "float* milliseconds;event_t start_event;event_t stop_event" EventElapsedTime)
```